src/main.js

```diff
@@ -336,19 +336,36 @@ class MiniGraphCard extends LitElement {
       </div>` : '';
   }
 
+  generateLegend(index) {
```

Comment:
There is not yet a function named `generateXYZ` in the code. My feeling is, we use `computeXYZ` a lot for generating text. Should we rename `generateLegend` to `computeLegend`?

Reply:
Renamed to computeLegend